docs/source/api_reference/models.rst

```diff
@@ -299,7 +299,7 @@ WGAN
 TextCNNModel
 ------------
 
-.. autoclass:: deepchem.models.TextCNNModel
```

Comment:
Don't remove the old TF model; add this as a new model

Reply:
Reverted replacing the old TF model. Added the docs for the torch model under the Torch Models section.